### PR TITLE
[s1 sim][new test cases] New s1 sim TCs to test PDN selection

### PR DESCRIPTION
**Pull request by pruthvihebbani**
Signed-off-by: Pruthvi Hebbani <pruthvi.hebbani@radisys.com>

[s1 sim][new test cases] New s1 sim TCs to test PDN selection

## Summary

This PR contains new s1 sim test cases to test PDN selection at agw

## Test Plan

- Verified s1 sim sanity

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_unknown_pdn_type.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            s1ap_types.tfwCmd.UE_ATTACH_ACCEPT_IND,
+            s1ap_types.ueAttachAccept_t,
+        )
+
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        # Send PDN Connectivity Request
+        apn = "ims"
+        # PDN Type 1 = IPv6, 2 = IPv6, 3 = IPv4v6
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_unknown_pdn_type.py]: typo `PDN Type 1 = IPv4`

File: lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        # Wait on EMM Information from MME
+        self._s1ap_wrapper._s1_util.receive_emm_info()
+
+        # Send PDN Connectivity Request
+        apn = "magma.ipv4"
+        self._s1ap_wrapper.sendPdnConnectivityReq(ue_id, apn)
+        # Receive PDN Connectivity reject
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_PDN_CONN_RSP_IND.value
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn.py:61]: how is this assert checking rejection?

Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_secondary_pdn_reject_multiple_sessions_not_allowed_per_apn.py:61]: In s1ap tester uePdnConRsp_t is a common message for handling Activate default EPS bearer context request/PDN connectivity reject

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**